content/docs/reference/spec/migration/platform-api-0.4-0.5.md

```diff
@@ -0,0 +1,47 @@
++++
+title="Platform API 0.4 -> 0.5"
++++
+
+<!--more-->
+
+This guide describes changes from platform API 0.4 to 0.5, who these changes are relevant for,
+as well as any actions that may be necessary.
+
```

Comment:
minor: This sentence looks like it would work as a good summary/snippet to describe the content.

```suggestion

This guide describes changes from platform API 0.4 to 0.5, who these changes are relevant for,
as well as any actions that may be necessary.

<!--more-->

```

Reply:
Since this sentence is basically duplicated for each guide, I felt that moving this into the summary would make the index a little weird. So I just moved one sentence into the index in 7129949 - what do you think?